File: 1192/1192.c
// bee 1192 - O jogo Matematico de Paula
// Alonso Martins
// 21/05/2024
#include <stdio.h>
#include <ctype.h>
#include <stdlib.h>

int main()
{
    int N, A, B; scanf("%d", &N);
    char n[5];

    while (N -- && scanf(" %s", n))
    {
        A = atoi(&n[0]), B = atoi(&n[2]);

        if (A == B)
            printf("%d\n", A * B);
        else if (isupper(n[1]))
            printf("%d\n", B - A);
        else
            printf("%d\n", A + B);
    }

    return 0;
}

File: 1192/1192.py
# bee 1192 - O jogo Matemático de Paula
# Alonso Martins
# 02/10/2024

def jogo_de_paula(x):
    a, b = map(int, x[::2])
    c = x[1]
    
    if a == b: return a * b
    if c.isupper(): return b - a
    return b + a


if __name__ == '__main__':
    n = int(input())
    
    for i in range(n):
        m = input()
        
        print(jogo_de_paula(tuple(m)))
